ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -891,3 +891,36 @@ def test_torch_leaky_relu_(
         negative_slope=alpha,
         test_values=False,
     )
+
+
+# hardswish
+@handle_cmd_line_args
+@given(
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.hardswish"
+    ),
+)
+def test_torch_hardswish(
+    dtype_and_input,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, input = dtype_and_input
+    assume("float16" not in input_dtype)
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_tree="nn.functional.hardswish",
+        input=np.asarray(input, dtype=input_dtype),
```

Comment:
Issue here seems to be that the frontend function requires this as a keyword variable.

The other possibility is the num-positional-args isn't working correctly

Reply:
I think the num_positional_args isn't working properly because all arguments are resulting into keyword args only.